### PR TITLE
Upgrade to SDK 2

### PR DESCRIPTION
**Pull request by ducanhkhu-ovo**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2016 OVO Energy
+   Copyright 2018 OVO Energy
```
Comment (joedarby) [inline LICENSE:190]: Not 2021? lol

**Review by joedarby — APPROVED** [review]

**Review by d-fernandes — COMMENTED** [review]
It looks good to me but I have very little context around this repo. What do we use it for ?

File: LICENSE (hunk 1 above)
Comment (ducanhkhu-ovo) [reply LICENSE:190]: the date is start date actually. I did this by mistake, will revert

**Comment by ducanhkhu-ovo** [comment]
> It looks good to me but I have very little context around this repo. What do we use it for ?

we use this throughout the platform when we need to deal with S3. Upload templates, attachments etc..

File: build.sbt [hunk 2]
```diff
@@ -123,7 +123,8 @@ lazy val common = (project in file("modules/common"))
   .settings(automateHeaderSettings(IntegrationTest))
   .settings(
     libraryDependencies ++= Seq(
-      "com.amazonaws" % "aws-java-sdk-core" % awsSdkVersion % Optional
+      "software.amazon.awssdk" % "auth" % awsSdkVersion,
+      "software.amazon.awssdk" % "sdk-core" % awsSdkVersion % Optional
```
Comment (ra-ovo) [inline build.sbt:127]: Oh wow, TIL: https://maven.apache.org/guides/introduction/introduction-to-optional-and-excludes-dependencies.html.

I had never bumped into this concept of optional dependency 🤦‍♂️

**Review by ra-ovo — APPROVED** [review]

**Pull request merged**